xarray/core/parallelcompat.py

```diff
@@ -132,8 +132,9 @@ def get_chunked_array_type(*args) -> ChunkManagerEntrypoint:
         if chunkmanager.is_chunked_array(chunked_arr)
     ]
     if not selected:
+        possible_missing_package_name = type(chunked_arr).__module__.split(".")[0]
         raise TypeError(
-            f"Could not find a Chunk Manager which recognises type {type(chunked_arr)}"
+            f"Could not find a Chunk Manager which recognises type {type(chunked_arr)}. Try installing the {possible_missing_package_name} package."
```

Comment:
```suggestion
            f"Could not find a Chunk Manager which recognises type {type(chunked_arr)}. Try searching for and installing a {possible_missing_package_name}_xarray package."
```

? Since you could have `cubed` installed but not `cubed-xarray`

Reply:
Fair point, but then would that be more confusing if the problem is that the user doesn't have Dask installed? Because then it would point them towards a non-existent `dask_xarray` package...